docs/index.md

```diff
@@ -21,7 +21,13 @@ gateway for multiple [Trino](https://trino.io) clusters.
 
 ## Other resources
```

Comment:
Related question for reviewers .. should we lift this whole section into a top level "Resources" page .. similar to the users page and therefore keep the front page shorter?

Reply:
Moved

Reply:
Thanks for confirming my suspicion..